contracts/AdjudicationFramework.sol

```diff
@@ -8,9 +8,8 @@ pragma solidity ^0.8.20;
 
 import {BalanceHolder} from "./lib/reality-eth/BalanceHolder.sol";
 
-import {IRealityETH} from "./interfaces/IRealityETH.sol";
-import {IArbitrator} from "./interfaces/IArbitrator.sol";
-import {IERC20} from "./interfaces/IERC20.sol";
+import {IRealityETH} from "./lib/reality-eth/interfaces/IRealityETH.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
We probably don't want this dependency when we move it back to the reality,eth repo so I will end up undoing this and putting the original thing back.

Reply:
Sure, that is fine. No worries